pybv/io.py

```diff
@@ -49,7 +49,7 @@ def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
         specified by ``unit``) are never scaled (e.g. ``'°C'``).
     sfreq : int | float
         The sampling frequency of the data.
-    ch_names : list of strings, shape (n_channels,)
+    ch_names : list of str, shape (n_channels,)
```

Comment:
unrelated by why do we keep writing "shape ..." for a list; it should only have a length…

Reply:
:+1: 

https://github.com/bids-standard/pybv/pull/78/commits/efbb1fc4de93b0262d943ed19cc0f3714cb2905a